com.protonvpn.www.metainfo.xml

```diff
@@ -3,130 +3,91 @@
-                Split tunneling support allows you to select which apps go through the VPN
-                tunnel
+                NetShield Ad-blocker: Protect yourself from ads, trackers,
+                and malware on websites and apps
             </li>
+            <li>VPN Accelerator: Unique technology increases VPN speeds by up to 400%</li>
         </ul>
-        <p>Why choose Proton VPN?</p>
+        <h5>Why Proton VPN?</h5>
         <ul>
-            <li>
-                No-logs VPN: We do not keep any logs that can identify what you do online
-            </li>
-            <li>
-                DNS and IPv6 leak protection: Our Linux app will never accidentally expose
-                your real IP address
-            </li>
-            <li>
-                Easy-to-use interface: The new Proton VPN Linux app has an intuitive
-                graphical user interface that lets you quickly activate features and connect
-                to servers
-            </li>
-            <li>
-                P2P support: Share and download files using BitTorrent and other P2P
-                protocols
-            </li>
-            <li>
-                VPN Accelerator: Free and available to everyone who uses Proton VPN, our
-                unique VPN Accelerator technology can improve speeds by over 400%
-            </li>
-            <li>
-                Strong encryption: Our Linux app uses the OpenVPN protocol at its strongest
-                encryption settings
-            </li>
-            <li>
-                Based in Switzerland: Switzerland has some of the world's strongest data
-                privacy laws
-            </li>
-            <li>
-                Secure streaming: Unblock popular streaming services and watch your favorite
-                shows
-            </li>
+            <li>No personal data required to sign up</li>
+            <li>Easy-to-use VPN apps for iOS, Android, MacOS, Windows, iPad, Linux, Apple TV, and Android TV</li>
+            <li>Our high-strength encryption provides a more secure connection than an internet proxy</li>
+            <li>Independently audited by third-party security experts</li>
+            <li>Trusted open-source code that anyone can review</li>
+            <li>Data protection using AES-256 and 4096 RSA encryption</li>
         </ul>
-        <p>Join the privacy revolution</p>
+        <h5>Join the privacy revolution</h5>
         <p>
-            Your support is important as it allows us to continue our mission to bring
-            online freedom to people around the globe. Get our private VPN free today and
-            enjoy fast VPN connections and a secure internet, from anywhere.
+            Your support helps us bring online freedom to people around the globe.
+            Join Proton today to protect yourself online and help create a secure,
+            private, and free internet for everyone.
         </p>
-        <p>Required dependencies</p>
```

Comment:
I've removed this since the dependencies are inside the flatpak so it does not really matter to have them here.

Reply:
OK will revert it back.